.github/workflows/bump-stable-snapshot.yml

```diff
@@ -0,0 +1,57 @@
+---
+name: update-stable-snapshot
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * *"
+
+permissions:
+  contents: read
+
+env:
+  JOB_URL: "${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
+
+jobs:
+  filter:
+    runs-on: ubuntu-latest
+    timeout-minutes: 1
+    outputs:
+      matrix: ${{ steps.generator.outputs.matrix }}
+    permissions:
+      contents: read
+    steps:
+      - id: generator
+        uses: elastic/oblt-actions/elastic/active-branches@v1
+        with:
+          exclude-branches: "7.17"
+          filter-branches: true
+
+  update-stable-snapshot:
+    runs-on: ubuntu-latest
+    needs: [filter]
+    permissions:
+      contents: write
+      pull-requests: write
+    strategy:
+      fail-fast: false
+      matrix: ${{ fromJson(needs.filter.outputs.matrix) }}
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ matrix.branch }}
```

Comment:
I think we can skip this, so we can always use the same `updatecli` pipeline from main, aka `.ci/updatecli/updatecli-bump-stack-version.yml`.

Or do you see a need to have different implementations per pipeline?

Reply:
The only difference I see is the branch name and the `.stable-snapshot-version` contents. The behaviour should be the same across all active branches